### ISSUE TITLE
Question about the loss function of Tf-reg KD 

### ISSUE THREAD

**Issue by HowieMa** [yuanli2333/Teacher-free-Knowledge-Distillation#24]
Hi, thank you for sharing such an awesome project. 
For the TF-reg KD, in [line 47 of my_loss_function.py](https://github.com/yuanli2333/Teacher-free-Knowledge-Distillation/blob/ecaa18475ebf657297fd340c5bdb312136b28313/my_loss_function.py#L47), should we also divide the temperature T on the output variable, like: 
`loss_soft_regu = nn.KLDivLoss()(F.log_softmax(outputs / T, dim=1), F.softmax(teacher_soft/T, dim=1))*params.multiplier `

As in Eq (9) of your paper, the loss is $$D_{KL}(p^d_\tau, p_\tau)$$

**Comment by DLoveS1314** [comment]
I am also very confused about this issue, looking forward to the author's answer
#19 answer your question
